java/client/src/test/java/glide/api/RedisClientTest.java

```diff
@@ -37,7 +39,12 @@ public void customCommand_success() throws ExecutionException, InterruptedExcept
         String cmd = "GETSTRING";
         CompletableFuture<Object> testResponse = mock(CompletableFuture.class);
         when(testResponse.get()).thenReturn(value);
-        when(commandManager.submitNewCommand(any(), any())).thenReturn(testResponse);
+        Command command =
+                Command.builder()
+                        .requestType(Command.RequestType.CUSTOM_COMMAND)
+                        .arguments(new String[] {cmd, key})
```

Comment:
varargs
```suggestion
                        .arguments(cmd, key)
```

Reply:
This is Command, that is removed from the other PR.  Let's just leave commandCommand changes to the other PR.